ApeWorX/ape-trezor
# refactor: import logic and beta

### What I did

Refactored to match the Ledger plugin structure
Updated version for beta release. 


fixes: #9 
While still waiting on release by Trezor, #8 is addressed by the PR by adding EIP-712 support
https://github.com/ApeWorX/ape/issues/327

### How I did it

Used the Ledger plugin for design/structure/layout to standardize hardware wallet plugins

### How to verify it

```console
$ ape trezor add test0
SUCCESS: Account '<address>' successfully added with alias 'test0'.

$ ape trezor add test1
SUCCESS: Account '<address>' successfully added with alias 'test1'.

$ ape trezor add test2
SUCCESS: Account '<address>' successfully added with alias 'test2'.

$ ape trezor list
<address> (alias: 'test0') (hd-path: 'm/44'/60'/0/0/0')
<address> (alias: 'test1') (hd-path: 'm/44'/60'/0/0/1')
<address> (alias: 'test2') (hd-path: 'm/44'/60'/0/0/2')

$ ape trezor delete test0
SUCCESS: Account 'test0' has been removed

$ ape trezor sign-message test1 "hello world"
<signature>

$ ape trezor verify-message "hello world" [signature]
signer: <address>

$ ape trezor delete-all
```

### Checklist
